conf/acas-base.env

```diff
@@ -959,4 +959,13 @@ CLIENT_CMPDREG_BULKLOADSETTINGS_DBPROPERTIES=[{'name': 'CAS Number', 'dataType':
 CLIENT_CMPDREG_SERVERSETTINGS_LIVEDESIGN_PREPROCESSORSETTINGS=
 
 # MAPS TO: client.cmpdreg.serverSettings.maxStandardizationDisplay
-CLIENT_CMPDREG_SERVERSETTINGS_MAXSTANDARDIZATIONDISPLAY=20000
\ No newline at end of file
+CLIENT_CMPDREG_SERVERSETTINGS_MAXSTANDARDIZATIONDISPLAY=20000
+
+# MAPS TO: client.version.version
+CLIENT_VERSION_VERSION=0.0.0
```

Comment:
Left this at semantic style versioning because I think we'll eventually go there with ACAS, this is just the default though incase its not set at build time.